src/main/scala/org/broadinstitute/dsde/firecloud/core/ProfileClient.scala

```diff
@@ -258,13 +258,13 @@ class ProfileClientActor(requestContext: RequestContext) extends Actor with Fire
 
   def checkUserInRawls(
     pipeline: WithTransformerConcatenation[HttpRequest, Future[HttpResponse]],
-    requestContext: RequestContext): Future[PerRequestMessage] = {
+    requestContext: RequestContext, userId: String): Future[PerRequestMessage] = {
```

Comment:
do I have to pass userId into checkUserInRawls? is there a way to pull it out of the requestContext? it's needed for the call to thurloe if the user is new, but i would assume that this id already exists somewhere in the request